src/utils/send-error.js

```diff
@@ -0,0 +1,62 @@
+require('dotenv').config();
+const { translateLanguage } = require('../languages/index');
+
+async function sendErrorToChannel(source, title, error, additionalInfo = {}) {
+  let client, commandName, user;
+
+  if (source && source.client && source.commandName) {
+    client = source.client;
+    commandName =
+      source.commandName ||
+      translateLanguage('sendChannelError.unknownCommand');
+    user = source.user
+      ? source.user.tag
+      : translateLanguage('sendChannelError.unknownUser');
+  } else {
+    client = source;
+    commandName =
+      additionalInfo.command ||
+      translateLanguage('sendChannelError.unknownFunction');
+    user = translateLanguage('sendChannelError.unknownUser');
+  }
+
+  additionalInfo.command = commandName;
```

Comment:
You deleted the aditionalInfo in the calls, so maybe we don't need additionalInfo?